src/main/java/com/psk/pyragas/ToolRent/usecases/Advertisements.java

```diff
@@ -44,11 +45,10 @@ public void init(){
     public String rentItem() {
         System.out.println("Rent was called");
 
-        //TODO:Code throws exception: attempt to create merge event with null entity.
         Order order = new Order();
+        order.setPrice(new BigDecimal(66));
```

Comment:
Can confirm that it does indeed insert an order with this price and a reference to the advertisement with id 1.